tests/decap/test_decap.py

```diff
@@ -119,51 +135,54 @@ def setup_teardown(request, duthosts, duts_running_config_facts, ip_ver, loopbac
     restore_default_decap_cfg(duthosts)
 
 
-def apply_decap_cfg(duthosts, ip_ver, loopback_ips, ttl_mode, dscp_mode, ecn_mode, op):
+def apply_decap_cfg(duthosts, ip_ver, get_ips, ttl_mode, dscp_mode, ecn_mode, op):
 
     decap_conf_template = Template(open("../ansible/roles/test/templates/decap_conf.j2").read())
 
     # apply test decap configuration (SET or DEL)
     for idx, duthost in enumerate(duthosts):
-        decap_conf_vars = {
-            'lo_ip': loopback_ips['lo_ips'][idx],
-            'lo_ipv6': loopback_ips['lo_ipv6s'][idx],
-            'ttl_mode': ttl_mode,
-            'dscp_mode': dscp_mode,
-            'ecn_mode': ecn_mode,
-            'op': op,
-        }
-        decap_conf_vars.update(ip_ver)
-        duthost.copy(
-            content=decap_conf_template.render(**decap_conf_vars),
-            dest='/tmp/decap_conf_{}.json'.format(op))
-
-        for asic_id in duthost.get_frontend_asic_ids():
-            swss = 'swss{}'.format(asic_id if asic_id is not None else '')
-            cmds = [
-                'docker cp /tmp/decap_conf_{}.json {}:/decap_conf_{}.json'.format(op, swss, op),
-                'docker exec {} swssconfig /decap_conf_{}.json'.format(swss, op),
-                'docker exec {} rm /decap_conf_{}.json'.format(swss, op)
-            ]
-            duthost.shell_cmds(cmds=cmds)
-        duthost.shell('rm /tmp/decap_conf_{}.json'.format(op))
+        ips = get_ips['ips']
+        ipv6s = get_ips['ipv6s']
+        if ips and ipv6s:
+            decap_conf_vars = {
+                'ip': get_ips['ips'][idx],
```

Comment:
Here `idx` is index of duthost. But list `get_ips['ips']` contains IP of all interfaces of all duthosts?